holopy/inference/prior.py

```diff
@@ -77,6 +86,25 @@ def __rmul__(self, value):
     def __truediv__(self, value):
         return self * (1/value)
 
+    def __rtruediv__(self, value):
+        return value * TransformedPrior(_reciprocal, self)
```

Comment:
Just curious -- why are these not `return TransformedPrior(operator.truediv, [self, value])` and `return TransformedPrior(operator.truediv, [value, self])`?

Reply:
They probably could be - previously the math behaviour had to be written manually so I tried to use identities to get everything onto `__mul__` and `__add__`. That's less of an issue now, but there is still specific handling for special values like 0 or 1. Do you think it would be more clear to use `operator` functions throughout?

Reply:
Personally, I think that keeping the same pattern throughout would be the clearest, but I'm not sure. For instance, if you are end up doing a lot of error handling, it could be cleaner to call the `__mul__` like you are doing here. 

It's up to you.